Formula/signal-cli.rb

```diff
@@ -1,8 +1,8 @@
 class SignalCli < Formula
   desc "CLI and dbus interface for WhisperSystems/libsignal-service-java"
   homepage "https://github.com/AsamK/signal-cli"
-  url "https://github.com/AsamK/signal-cli/releases/download/v0.9.0/signal-cli-0.9.0.tar.gz"
-  sha256 "c24f2493e3c6d27c36384ee671c1a33f8df9484cad4ad472d6e9f183a12a3fff"
+  url "https://github.com/AsamK/signal-cli/archive/refs/tags/v0.10.4.2.tar.gz"
```

Comment:
Looks like previous URL is now OS-specific. Not sure if we have a way to express this in Homebrew Formula DSL:
```
https://github.com/AsamK/signal-cli/releases/download/v0.10.4.2/signal-cli-0.10.4.2-macOS.tar.gz
https://github.com/AsamK/signal-cli/releases/download/v0.10.4.2/signal-cli-0.10.4.2-Linux.tar.gz
```

Alternatively can try building from source

Reply:
Given that we do rebuild the `libsignal` shared library, using either tarball may actually work.

For now, just switched to source build.